tiny_obj_loader.h

```diff
@@ -219,19 +219,42 @@ typedef struct {
   int texcoord_index;
 } index_t;
 
-typedef struct {
+// mesh_t will contain faces corresponsing to a smoothing group id.
+struct mesh_t{
+  int smoothingGroupId;                          // Smoothing group id. 
```

Comment:
New proposal: smoothingGroupId is stored in mesh_t. 
A mesh_t will contain faces corresponding to a smoothing group id.